CHANGELOG.md

```diff
@@ -8,6 +8,17 @@ and this project adheres to
 
 ## [Unreleased]
 
+- **Breaking**: Support linting of sources.
```

Comment:
That's a feature, not a breaking change, right?

Reply:
Yeah, I'll fix that